src/data/index.ts

```diff
@@ -0,0 +1,32 @@
+
+const data = {
+    "contactInfo": {
+        "fullName": "Saul Lopez",
+        "emailAddress": ""
+    },
+    "github": {
+        "profileURL": "https://github.com/saullopezc",
+        "username": "saullopezc"
+    },
+    "credentials": {
+        "password": ""
+    },
+    "personalInfo": {
+        "questions": [
+            {
+                "question": "If I was a Sr. Programmer, I would like to build:",
+                "answer": "Paquetes o librerias "
+            },
+            {
+                "question": "Por favor indica el URL que me lleva a la línea de código de la definición de React.useEffect",
+                "answer": "https://es.reactjs.org/docs/hooks-reference.html#useeffect"
+            },
+            {
+                "question": "code is poetry, because:",
+                "answer": "Por la expresion y la belleza del codigo"
```

Comment:
```suggestion
                "answer": "Por la expresión y la belleza del código"
```

hay que ser cuidadosos con la ortografía. 
los detalles son importantes.